src/main/java/com/exasol/adapter/installer/File.java

```diff
@@ -3,38 +3,38 @@
 import java.nio.file.Path;
 
 /**
- * Represents a JAR file.
+ * Represents a file.
  */
-public class JarFile {
+public class File {
```

Comment:
can't we use the Java standard `File` class from the `io` library?

Reply:
Well, this is a class-wrapped for two strings. A standard class provides too much unnecessary methods